bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,202 @@ class CoffeeLintBear:
+                         apply_no_trailing_whitespaces_in_empty_line: bool=True,
+                         no_unnecessary_double_quotes: bool=True,
+                         use_english_operator: bool=False,
+                         space_around_operators: bool=True,
+                         space_after_comma: bool=True):
+        """
+        :param: max_line_length:
+            Maximum number of characters per line.
+        :param: apply_max_line_length_on_comments:
+            Determines if ``max_line_length`` should also affects comments or
+            not.
+        :param: space_before_and_after_arrow:
+            Determines if spaces should be used before and/or after the arrow.
+        :param: braces_spacing:
+            Checks if the proper spacing is used inside curly braces.
+        :param: braces_spacing_width:
+            Determines the number of white space after the opening ``{`` and
+            before the closing brace ``}`` given that there is something within
+            the braces.
+        :param: spacing_in_empty_braces:
+            Determines the number of white space after the opening ``{`` and
+            before the closing brace ``}`` given an empty content.
+        :param: camelcase_classes_name:
+            Checks whether the classes name should are in camel-case or not.
+        :param: spaces_before_and_after_colon:
+            Determines if spaces should be added before and after colons.
+        :param: spaces_before_colon:
+            Determines the number of space before colon when
+            ``spaces_before_and_after_colon`` is `` True``.
+        :param: spaces_after_colon:
+            Determines the number of space after colon when
+            ``spaces_before_and_after_colon`` is `` True``.
+        :param: end_file_with_a_single_newline:
+            Checks if the file ends with a single newline.
+        :param: tab_width:
+            Number of spaces for indentation.
+        :param: no_tabs
+            Forbids tabs in indentation.
+        :param: newlines_after_classes:
+            Checks if there is number equal ``number_of_lines_after_classes`` of
+            lines after classes definition.
+        :param: number_of_lines_after_classes:
+            Determines the number of lines that seperate the a class definition
+            and the rest of the code.
+        :param: no_backticks:
+
+        :param: no_implicit_braces:
+            Prohibits implicit braces when declaring object literals.
+
+            Example: If ``no_implicit_braces = True`` then
+            ```
+            1:2, 3:4
+            ``` is prohibited whereas
+            ```
+            {1:2, 3:4}
+            ``` is accepted.
+        :param: no_function_call_without_parentheses:
+            prohibits implicit parentheses on function calls.
+        :param: no_interpolation_in_single_quotes:
+            Prohibits string interpolation in a single quoted string.
```

Comment:
yeah coffee dev knows what it means
``#{string}`` is an example of interpolation

2016-06-25 7:28 UTC+01:00, Abhay Raizada <notifications@github.com>:
>> +        :param: no_backticks:
>> +
>> +        :param: no_implicit_braces:
>> +            Prohibits implicit braces when declaring object literals.
>> +
>> +            Example: If ``no_implicit_braces = True`` then
>> +            ```
>> +            1:2, 3:4
>> +            ``` is prohibited whereas
>> +            ```
>> +            {1:2, 3:4}
>> +            ``` is accepted.
>> +        :param: no_function_call_without_parentheses:
>> +            prohibits implicit parentheses on function calls.
>> +        :param: no_interpolation_in_single_quotes:
>> +            Prohibits string interpolation in a single quoted string.
>
> no idea what interpolation means, does it make sense to coffee dev?
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/coala-analyzer/coala-bears/pull/555/files/fc86d577d11a01dd6e49ee88d3d6eaf82b80c860#r68485038